catalog/dags/es/create_new_es_index/utils.py

```diff
@@ -0,0 +1,53 @@
+def merge_configurations(base_configuration, update_configuration):
```

Comment:
I could -- I was planning on just pulling things out as they're needed by the new DAGs rather than trying to anticipate exactly what would be needed here. This is an example of something I wouldn't have guessed we would use for the filtered index DAGs, actually -- maybe the tasks for triggering the reindex, but I didn't think we had any plans for merging configurations elsewhere.